examples/closure-app/common/closure-control/src/ClosureControlMain.cpp

```diff
@@ -56,16 +56,18 @@ CHIP_ERROR ClosureControlInit(EndpointId endpointId)
         return CHIP_ERROR_INCORRECT_STATE;
     }
 
-    gClosureCtrlDelegate = std::make_unique<ClosureControlDelegate>(endpointId);
+    gClosureCtrlDelegate = std::make_unique<ClosureControlManager>();
     if (!gClosureCtrlDelegate)
     {
-        ChipLogError(AppServer, "Failed to allocate memory for ClosureControlDelegate");
+        ChipLogError(AppServer, "Failed to allocate memory for ClosureControlManager");
         return CHIP_ERROR_NO_MEMORY;
     }
 
     /* Manufacturer may optionally not support all features, commands & attributes */
-    gClosureCtrlInstance = std::make_unique<ClosureControlInstance>(EndpointId(endpointId), *gClosureCtrlDelegate,
-                                                                    Feature::kCalibration, OptionalAttribute::kCountdownTime);
+    gClosureCtrlInstance = std::make_unique<ClosureControlInstance>(
+        EndpointId(endpointId), *gClosureCtrlDelegate,
+        BitMask<Feature, uint32_t>(Feature::kCalibration, Feature::kPositioning, Feature::kSpeed, Feature::kMotionLatching),
```

Comment:
To make the app easier to understand / moidfy, we have section within this file that has the configuration as constexpr; Not directly in the constructor.